test/scales/timeScaleTests.ts

```diff
@@ -1,178 +1,256 @@
+        let tenMinuteTicks = scale.tickInterval(Plottable.TimeInterval.minute, 10);
+        assert.strictEqual(tenMinuteTicks.length, 7, "generated correct number of ticks for 1 hour span, every 10 minutes");
+        // 1 minute span
+        scale.domain([new Date(2000, 0, 1, 0, 0, 0, 0), new Date(2000, 0, 1, 0, 1, 0, 0)]);
+        let secondTicks = scale.tickInterval(Plottable.TimeInterval.second);
+        assert.strictEqual(secondTicks.length, 61, "generated correct number of ticks for 1 minute span, every second");
+      });
     });
 
-  it("can be padded", () => {
-    let scale = new Plottable.Scales.Time();
-    scale.padProportion(0);
-    let unpaddedDomain = scale.domain();
-    scale.addIncludedValuesProvider((scale: Plottable.Scales.Time) => unpaddedDomain);
-    scale.padProportion(0.1);
-    assert.operator(scale.domain()[0].getTime(), "<", unpaddedDomain[0].getTime(), "left side of domain was padded");
-    assert.operator(scale.domain()[1].getTime(), ">", unpaddedDomain[1].getTime(), "right side of domain was padded");
-  });
+    describe("Auto Domaining", () => {
+      let scale: Plottable.Scales.Time;
 
-  it("respects padding exceptions", () => {
-    let scale = new Plottable.Scales.Time();
-    let minValue = new Date(2000, 5, 4);
-    let maxValue = new Date(2000, 5, 6);
-    scale.addIncludedValuesProvider((scale: Plottable.Scales.Time) => [minValue, maxValue]);
-    scale.padProportion(0.1);
-    assert.operator(scale.domain()[0].getTime(), "<", minValue.getTime(), "left side of domain is normally padded");
-    assert.operator(scale.domain()[1].getTime(), ">", maxValue.getTime(), "right side of domain is normally padded");
-    scale.addPaddingExceptionsProvider(() => [minValue]);
-    assert.strictEqual(scale.domain()[0].getTime(), minValue.getTime(), "left side of domain isn't padded if it matches the exception");
-    scale.addPaddingExceptionsProvider(() => [maxValue]);
-    assert.strictEqual(scale.domain()[1].getTime(), maxValue.getTime(), "right side of domain isn't padded if it matches the exception");
-  });
+      beforeEach(() => {
+        scale = new Plottable.Scales.Time();
+      });
 
-  it("autoDomain() expands single value to [value - 1 day, value + 1 day]", () => {
-    let scale = new Plottable.Scales.Time();
-    scale.padProportion(0);
-    let singleValue = new Date(2000, 5, 5);
-    let dayBefore = new Date(2000, 5, 4);
-    let dayAfter = new Date(2000, 5, 6);
-    scale.addIncludedValuesProvider((scale: Plottable.Scales.Time) => [singleValue, singleValue]);
-    scale.autoDomain();
-    let domain = scale.domain();
-    assert.strictEqual(domain[0].getTime(), dayBefore.getTime(), "left side of domain was expaded by one day");
-    assert.strictEqual(domain[1].getTime(), dayAfter.getTime(), "right side of domain was expaded by one day");
-  });
+      it("expands single value domains to [value - 1, value + 1] when auto domaining", () => {
```

Comment:
autodomain()-ing
according to the best practice, use parentheses when referring to method name